entrypoint.sh

```diff
@@ -27,6 +27,18 @@ elif [ "$MODE" == "worker" ] ; then
     /opt/spark/sbin/start-worker.sh --cores "$CORES" --memory "$MEM" "$SPARK_MASTER_URL"
 elif [ "$MODE" == "bash" ] ; then
     bash
+elif [ "$MODE" == "notebook" ] ; then
+    echo "starting jupyter notebook"
+    WORKSPACE_DIR="/cdm_shared_workspace"
+    mkdir -p "$WORKSPACE_DIR" || { echo "Error: Failed to create workspace directory"; exit 1; }
+    cd "$WORKSPACE_DIR" || { echo "Error: Failed to navigate to workspace directory"; exit 1; }
```

Comment:
Do we really need the error handling here? This should never happen

Reply:
removed.